From 829a6eb9abfe71ea80db7fd09c56e26e42a5552e Mon Sep 17 00:00:00 2001
From: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
Date: Fri, 22 Jan 2016 12:39:33 +0530
Subject: [PATCH 4/7] USBC/FUSB: Avoid fusb pd reset during hardreset.

Fusb pd-reset while triggering hardreset is causing incomplete
hardreset as pd-reset resets the fusb hardware. Hence pd-reset
can be skipped while initiating hardreset as this will be done
on hardreset complete.

Change-Id: I02c78eb9bc09d026aa2e0bcc59acdcc9a7f7864a
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-17109
Signed-off-by: Venkataramana Kotakonda <venkataramana.kotakonda@intel.com>
Reviewed-on: https://android.intel.com:443/461212
---
 drivers/usb/typec/phy-fusb300.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/typec/phy-fusb300.c b/drivers/usb/typec/phy-fusb300.c
index ff1d727..3583937 100644
--- a/drivers/usb/typec/phy-fusb300.c
+++ b/drivers/usb/typec/phy-fusb300.c
@@ -1200,8 +1200,6 @@ static int fusb300_phy_reset(struct typec_phy *phy)
 	else
 		fusb302_pd_send_hard_rst(phy);
 	mutex_unlock(&chip->lock);
-	/* Reset the fusb tranceiver */
-	fusb300_reset_pd(phy);
 	return 0;
 }
 
-- 
1.9.1

